PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+Please complete the following checklist:
+
+* [ ] I have read the guidelines in [CONTRIBUTING.md](CONTRIBUTING.md).
+
+* [ ] I have documented all public-facing changes in [CHANGELOG.md](CHANGELOG.md).
+
+* [ ] **This pull request includes code, documentation, or other content taken from external source(s).** If this is the case, ensure the external source's license is compatible with scikit-bio's [license](COPYING.txt). Include the license in the `licenses` directory and add a comment in the code giving proper attribution. Ensure any other requirements set forth by the license and/or author are satisfied. **It is your responsibility to disclose code, documentation, or other content taken from external source(s).** If you have questions about whether something can be included in the project or how to give proper attribution, include those questions in your pull request and a reviewer will assist you.
+
+* [ ] **This pull request does not include code, documentation, or other content taken from external source(s).**
```

Comment:
So that the user is forced to check one box or the other, making an explicit statement about derived content. This is a similar strategy to the bioconda example you posted above.